app/views/bourbons/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>List of Bourbons</h1>
+<ul>
+<% @bourbons.each do |bourbon| %>
+<li>
+  <%= link_to bourbon.name, bourbon_path(bourbon)%> - <%=bourbon.proof%> <%=bourbon.distillery%> - <%=bourbon.varietal%>
+</li>
+<% end %>
+</ul>
+
+<%= link_to "Add Bourbon", new_bourbon_path %>
+<br>
```

Comment:
Don't use `<br>` tags! CSS should be used to create space between elements. More [here](http://line25.com/articles/youre-doing-it-wrong-common-html-tag-misuse).